docs/sovereign/setup.md

```diff
@@ -45,100 +43,94 @@ Before proceeding, ensure that a SSH key for GitHub is configured on your machin
+- **mx-chain-proxy-go**: Repository for the proxy.
+- **mx-chain-sovereign-bridge-go**: Repository for the cross-chain service.
 :::
 
-## Step 4: Deploy Contracts on Testnet
+## Step 3: Deploy Sovereign setup
 
 First navigate to the sovereignBridge folder:
 
 ```bash
-cd ./sovereignBridge
+cd sovereignBridge
 ```
 
-1. Update the `/config/configs.cfg` file with the Owner and WASM file details. Example:
+1. Update the configuration file `config/configs.cfg` with paths you want to use, wallet location and main chain constants. Example:
     ```ini
+    # Sovereign Paths
+    SOVEREIGN_DIRECTORY="~/sovereign"
+    TXS_OUTFILE_DIRECTORY="${SOVEREIGN_DIRECTORY}/txsOutput"
+    CONTRACTS_DIRECTORY="${SOVEREIGN_DIRECTORY}/contracts"
+
     # Owner Configuration
-    WALLET="~/wallet.pem" 
+    WALLET="~/wallet.pem"
 
     # Main Chain Constants
     PROXY = https://testnet-gateway.multiversx.com
     CHAIN_ID = T
-
-    # WASM Files
-    ESDT_SAFE_WASM="~/contracts/esdt-safe.wasm"
-    FEE_MARKET_WASM="~/contracts/fee-market.wasm"
-    MULTISIG_VERIFIER_WASM="~/contracts/multisigverifier.wasm"
     ```
 
 :::note
 
+- **SOVEREIGN_DIRECTORY, TXS_OUTFILE_DIRECTORY, CONTRACTS_DIRECTORY** - represent the paths to the location where the deploy scripts will generate the outputs.
 - **WALLET** - should represent the wallet generated at Step 1.
 - **PROXY** - in this case, for the purpose of the test, the used proxy is the testnet one. Of course that the proper proxy should be used when deploying your own set of contracts depending on the development phase of your project.
 - **CHAIN_ID** - should represent the chain ID of the chain where the contracts are to be deployed. The currently supported constants are:
     - **"1"** for Mainnet;
     - **"D"** for Devnet;
     - **"T"** for Testnet;
-- **ESDT_SAFE_WASM, FEE_MARKET_WASM, MULTISIG_VERIFIER_WASM** - represent the paths to the location where the contracts have been downloaded at Step 2.
-
-Before going forward, please make sure you have funds in the wallet on the chain you chose in the configuration file.
 :::
 
 2. Source the script:
     ```bash
     source script.sh
     ```
 
-3. Deploy all bridge contracts, Sovereign Notifier and update Sovereign Configs:
+3. Install the [software dependencies](/sovereign/software-dependencies) and download the cross-chain contracts by running the Sovereign bridge prerequisites:
     ```bash
-    deployMainChainContractsAndSetupObserver
+    runBridgePrerequisites
     ```
 
-4. Initialize and deploy sovereign chain with all required services:
+4. Deploy all cross-chain contracts on main chain and deploy sovereign chain with all dependent services:
     ```bash
-    sovereignDeploy
+    deploySovereignWithCrossChainContracts
     ```
 
 :::note
-## Alternatively, you can do everything manually step-by-step:
-### Deploy contracts:
-```bash
+## Alternatively, you can deploy everything manually step-by-step:
```

Comment:
We should delete this part, I don't think nobody will ever run these one by one and furthermore, since the development is heavily changing, we need to always maintain this part here without much benefits.

Reply:
This was good because some people want to run it step by step to understand the steps. Also, this was helpful for debugging if not everything was running properly.

Reply:
deleted the note